src/wasm_runner.rs

```diff
@@ -10,6 +10,39 @@ use crate::wasm_module::WasmModuleSource;
 
 const STDERR_FILE: &str = "module.stderr";
 
+#[derive(Clone, Default)]
+pub struct WasmLinkOptions {
+    pub http_allowed_hosts: Option<Vec<String>>,
+    pub http_max_concurrency: Option<u32>,
+}
+
+impl WasmLinkOptions {
+    pub fn none() -> Self {
+        Self::default()
+    }
+
+    pub fn with_http(
```

Comment:
This is arguably YAGNI stuff for a future where there may be multiple things that we want to link in.  Trying to avoid big future constructor lists.  But maybe we should leave that until the constructor list becomes impractical.

Reply:
A possible issue with this approach is that you cannot easily construct an objects option with multiple invocations of `with_*` functions — we're pretty much going back to the builder pattern here.
Again, I don't have strong opinions here — feel free to leave this as is, and we can explore it as we add more linker options.